gpii/node_modules/userListeners/src/listeners.js

```diff
@@ -111,6 +111,8 @@ fluid.defaults("gpii.userListener", {
     members: {
         // Set to true to call proximityTriggered, otherwise login/logout.
         proximity: false,
+        // Indicates that the logins by this listener are environmental logons (such as auto-logons) [GPII-3988]
+        environmental: false,
```

Comment:
The flag name handled by the lifecycle Manager is called ["isEnvironmentalLogin"](https://github.com/GPII/universal/blob/master/gpii/node_modules/lifecycleManager/src/LifecycleManager.js#L354) rather than "environmental".

Reply:
Fixed.